build/ps-entrypoint.sh

```diff
@@ -288,10 +294,10 @@ if [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			CREATE USER 'operator'@'${MYSQL_ROOT_HOST}' IDENTIFIED BY '${OPERATOR_ADMIN_PASSWORD}' ;
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
-			CREATE USER 'xtrabackup'@'localhost' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
-			GRANT SYSTEM_USER, BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'xtrabackup'@'localhost';
-			GRANT SELECT ON performance_schema.log_status TO 'xtrabackup'@'localhost';
-			GRANT SELECT ON performance_schema.keyring_component_status TO 'xtrabackup'@'localhost';
+			CREATE USER 'xtrabackup'@'%' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
```

Comment:
Does it need to be 'xtrabackup'@'%' or it can stay 'xtrabackup'@'localhost' if we are running in a sidecar? We should use least privileges.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/482157b23ae53ce96ed6d590feca38715a46007d